boards/sodaq-one/include/board.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2017 Kees Bakker, SODAQ
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sodaq-one
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the SODAQ ONE board
+ *
+ * @author      Kees Bakker <kees@sodaq.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    xtimer configuration
+ * @{
+ */
+#define XTIMER_WIDTH        (16)
+/** @} */
+
+/**
+ * @name    LED pin definitions and handlers
+ * @{
+ */
+#define LED_GREEN_PIN       GPIO_PIN(PB, 10)
```

Comment:
The LED macro should be called LED_0_XX/LED_1_XX, etc to be consistent with other boards. This way those macros can be used on any platform that provides them.